### DISCUSSION TITLE
How can I Pull Request

### DISCUSSION THREAD

**Discussion by AdarshPradhan1508** [fineanmol/hacktoberfest#796] [Q&A]
i have made all the changes in my local pc after cloning this repository for hacktoberfest2022 what to do next i am unable to pull request . Pls give me the solution.

**Comment by Drako-Rexon** [comment]
I have the same issue

**Comment by Thre4dripper** [comment]
- Step-1
> Fork the repo in your account
- Step-2
> Clone it to your PC and add your contributions in it
- Step-3
> When you push it to your github profile, then an option "create a pull request" will appear in your copy of this repo, just click on it.
- Step-4
> Write the necessary details in the comment of your PR and click create pull request, and its done

When a maintainer see your PR, he can merge your changes into this repo.

**Reply by AdarshPradhan1508** [reply]
Thank you so much
